src/command_modules/azure-cli-role/azure/cli/command_modules/role/_params.py

```diff
@@ -21,7 +21,9 @@ def load_arguments(self, _):
     with self.argument_context('ad') as c:
         c.argument('_subscription')  # hide global subscription param
         c.argument('owner_object_id', help="owner's object id")
-        c.argument('not_mine', action='store_true', help='do not set the current user as the owner', arg_group='Ownerships')
+        c.argument('mine', action='store_true', help='list entities owned by the current user')
```

Comment:
Are we set with `--mine`? Sounds a little strange, when we can do something like `--user`

Reply:
I can go either way. `--user` would make the argument appear to accept a value, but there is really no value to supply
@tjprescott, thoughts?

Reply:
I tend to agree that where `--user` is used elsewhere it takes a value. Perhaps `--show-mine`?

Reply:
Some futher suggestions: `--my-user` or `--current-user`

Reply:
I will go with `--show-mine` as a verb looks better for a flag.